.github/workflows/run-since-updater.yml

```diff
@@ -0,0 +1,36 @@
+name: Run update-since-todo.py
+
+on:
+  schedule:
+    - cron: "0 16 * * WED"
```

Comment:
Safer to do this Thursdays, since Wednesdays is when we publish weekly security updates.

Reply:
Indeed, good catch.